include/manif/impl/se3/SE3_base.h

```diff
@@ -206,6 +231,43 @@ SE3Base<_Derived>::translation() const
   return coeffs().template head<3>();
 }
 
+template <typename _Derived>
+void SE3Base<_Derived>::quat(const SE3Base<_Derived>::QuaternionDataType& quaternion)
```

Comment:
I believe the following is enough,
```suggestion
void SE3Base<_Derived>::quat(const QuaternionDataType& quaternion)
```